### PR TITLE
Specify network on compile command

### PR DESCRIPTION
**Pull request by vsavovlime**

### CODE REVIEW

File: commands.js [hunk 1]
```diff
@@ -31,10 +32,11 @@ const addInitOption = (program) => {
 const addCompileOption = (program) => {
   program
     .command('compile')
-    .option('--path [path]', 'Path to contract files', './contracts')
+    .option('-u --url [hostname]', 'Node to connect to', config.localhost)
```
Comment (maptuhec) [inline commands.js]: rename the option to --network, and the argument passed to network also

Comment (vsavovlime) [reply commands.js]: Network might be used to specify local,ropsten,rinkeby etc, **url** is more suitable?

**Comment by maptuhec** [comment]
@vsavovlime  Have you added unit tests for this?

File: commands.js (hunk 1 above)
Comment (vsavovlime) [reply commands.js]: nodeUrl sounds great

File: utils.js [hunk 2]
```diff
@@ -78,17 +78,22 @@ const getFiles = async function (directory, regex) {
...
-      process,
-      keypair: config.keyPair,
-      internalUrl: config.internalHost,
-      forceCompatibility: true,
-      nativeMode: true
-    })
+    client = await Universal(
+      { url: url, 
+        process, 
+        keypair: config.keyPair,
```
[the remark this answers is not in the document]
Comment (vsavovlime) [reply utils.js]: Done

**Pull request merged**